packages/mdc-data-table/_mixins.scss

```diff
@@ -0,0 +1,244 @@
+      @include mdc-data-table-cell-padding(
+        $leading-padding: $mdc-data-table-cell-leading-padding,
+        $trailing-padding: $mdc-data-table-cell-trailing-padding,
+        $query: $query
+      );
+    }
+
+    display: inline-flex;
+    flex-direction: column;
+  }
+
+  .mdc-data-table__table {
+    width: 100%;
+    border: 0;
+    white-space: nowrap;
+    border-collapse: collapse;
+  }
+
+  .mdc-data-table__cell {
+    @include mdc-typography(body2, $query: $query);
+  }
+
+  .mdc-data-table__cell--numeric {
+    text-align: right;
+  }
+
+  .mdc-data-table__header-cell--checkbox,
+  .mdc-data-table__cell--checkbox {
+    padding-right: 0;
+  }
+
+  .mdc-data-table__header-cell {
+    @include mdc-typography(subtitle2, $query: $query);
+
+    @include mdc-feature-targets($feat-structure) {
+      text-align: left;
+
+      @include mdc-rtl {
+        /* @noflip */
+        text-align: right;
+      }
+    }
+  }
+}
+
+@mixin mdc-data-table-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  @include mdc-feature-targets($feat-color) {
+    @include mdc-theme-prop("background-color", $color);
+  }
+}
+
+@mixin mdc-data-table-header-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-data-table__header-row {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-data-table-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-data-table__row {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-data-table-selected-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-data-table__row--selected {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-data-table-checked-icon-color($color, $query: mdc-feature-all()) {
+  .mdc-data-table__header-row-checkbox,
+  .mdc-data-table__row-checkbox {
+    @include mdc-states($color, $query: $query);
+    @include mdc-checkbox-container-colors($marked-stroke-color: $color, $marked-fill-color: $color, $query: $query);
+    @include mdc-checkbox-focus-indicator-color($color, $query: $query);
+  }
+}
+
+@mixin mdc-data-table-checkbox-touch-dimension($dimension, $query: mdc-feature-all()) {
+  .mdc-data-table__header-row-checkbox,
+  .mdc-data-table__row-checkbox {
+    @include mdc-checkbox-touch-dimension($dimension, $query: $query);
+  }
+}
+
+@mixin mdc-data-table-divider-color($color, $query: mdc-feature-all()) {
+  .mdc-data-table__row {
+    border-top-color: $color;
+  }
+}
+
+@mixin mdc-data-table-divider-size($size, $query: mdc-feature-all()) {
```

Comment:
it looks like you're not using $query in here or the mixin above/below

Reply:
Good catch. Sent PR 4891 to fix feature targeting for all mixins.